kf/processCheck.go

```diff
@@ -59,8 +59,25 @@ func (c *pCheck) pMonitorWorker(bpfProgs map[string]*list.List, direction string
 					bpf.RestartCount++
 					log.Warn().Msgf("pMonitor BPF Program is not running. Restart attempt: %d, program name: %s, iface: %s",
 						bpf.RestartCount, bpf.Program.Name, ifaceName)
-					if err := bpf.Start(ifaceName, direction, c.Chain); err != nil {
-						log.Error().Err(err).Msgf("pMonitor BPF Program start failed for program %s", bpf.Program.Name)
+					//  User program is a daemon and not running, but kernel program is loaded
+					if !userProgram && kernelProgram {
+						if err := bpf.StartUserProgram(ifaceName, direction, c.Chain); err != nil {
+							log.Error().Err(err).Msgf("pMonitor BPF Program start user program failed for program %s", bpf.Program.Name)
```

Comment:
what does "pMonitor" mean?

Reply:
pMonitor is to monitor bpf programs, initially it was monitoring only user program, now checks for bpf program is loaded and active.

Reply:
I don't follow.  Is it a Linux command?  is it some function in the l3afd repo?  Or what?

Reply:
Apparently it should say pMonitorWorker in processCheck.go
(not pMonitor).